### ISSUE TITLE
Missing padding 

### ISSUE THREAD

**Issue by mainrs** [catppuccin/gtk#156]
**Describe the bug**
The tab buttons at the top of programs have no padding. I re-installed the themes using `python install.py all -a all`. Before that, the tabs and panel buttons had larger padding on them.

**Mention your distribution and your theme version**
Latte Mauve Light

**Select how you installed the theme**
- [x] GitHub Releases
- [ ] AUR package 
- [ ] Nixpkg
- [ ] Using the python install script 
- [ ] Flatpak

**Symlinked the gtk-4.0 folder?**
Yes

**Expected behavior**
There should be some padding around the tab buttons and panel items.
![image](https://github.com/catppuccin/gtk/assets/5113257/793a6e51-f56c-4aad-b249-063fef330346)

**Screenshots**
![image](https://github.com/catppuccin/gtk/assets/5113257/e975415c-19d6-4e1a-bb89-1d984f4822cf)
![image](https://github.com/catppuccin/gtk/assets/5113257/21685660-e99f-4833-986c-f476eb547d93)

**Additional context**
Please mention any additional context that might be applicable.

**Comment by rubyowo** [comment]
I don't... see the issue?

**Comment by mainrs** [comment]
The first screenshot is how it should look. The second one of the add/remove application in manjaro has no padding on the tab icons.

**Comment by Syndrizzle** [comment]
Are you on a HiDPi Display?

**Comment by mainrs** [comment]
yes, I am. And it seems that this is the source of the issue. The UI looks completely differently once I change the settings. I guess this is not specific to the Catppuccin theme. 

Concerning the screenshot not being the right theme. You are right. I had to fully reboot to apply the theme. A logout was not enough. But the padding did not change. Only the colors (as you noticied @TGRush).

**Comment by rubyowo** [comment]
Can you confirm the padding issue happens in Colloid GTK as well?

**Comment by nullishamy** [comment]
I don't see a problem with the images provided, but if you would like to test with the latest release (https://github.com/catppuccin/gtk/releases/tag/v1.0.0-rc3), that would be good.
